packages/sync/src/class-main.php

```diff
@@ -15,18 +17,6 @@ class Main {
 	 * Initialize the main sync actions.
 	 */
 	public static function init() {
-		// Check for WooCommerce support.
-		add_action( 'plugins_loaded', array( 'Automattic\\Jetpack\\Sync\\Actions', 'initialize_woocommerce' ), 5 );
-
-		// Check for WP Super Cache.
-		add_action( 'plugins_loaded', array( 'Automattic\\Jetpack\\Sync\\Actions', 'initialize_wp_super_cache' ), 5 );
-
-		/*
-		 * Init after plugins loaded and before the `init` action. This helps with issues where plugins init
-		 * with a high priority or sites that use alternate cron.
-		 */
-		add_action( 'plugins_loaded', array( 'Automattic\\Jetpack\\Sync\\Actions', 'init' ), 90 );
-
```

Comment:
Moving these into the Jetpack class makes the Sync package even more dependent on Jetpack. I'm not sure it that's a concern for this package.